lib/Auth/Source/Globus.php

```diff
@@ -118,6 +123,24 @@ protected function getFullName($name) {
             'first_name' => 'UNKNOWN',
             'last_name' => 'UNKNOWN'
         );
+
+        /*
```

Comment:
this doesnt need to be in the code, this should be a unit test, but could just be a comment on the PR